src/background/extension-details-view-controller.ts

```diff
@@ -52,20 +54,23 @@ export class ExtensionDetailsViewController implements DetailsViewController {
     };
 
     private hasUrlChange(changeInfo: chrome.tabs.TabChangeInfo, targetTabId): boolean {
-        return (
-            changeInfo.url != null &&
-            !changeInfo.url
-                .toLocaleLowerCase()
-                .includes(this.getDetailsUrlWithExtensionId(targetTabId).toLocaleLowerCase())
-        );
+        if (changeInfo.url == null) {
```

Comment:
This function's logic hasn't changed, I just refactored it with clearer names for the intermediates so I could set breakpoints during debugging and didn't change it back after because I thought it looked more clear this way.